nox/sessions.py

```diff
@@ -184,6 +185,13 @@ def create_tmp(self) -> str:
         self.env["TMPDIR"] = tmpdir
         return tmpdir
 
+    @property
+    def cache_dir(self) -> pathlib.Path:
+        """Create and return a 'shared cache' directory to be used across sessions."""
+        path = os.path.join(self._runner.envdir, ".shared")
+        os.makedirs(path, exist_ok=True)
+        return pathlib.Path(path)
```

Comment:
Now if everything else is a `pathlib.Path` all you have to do is

```python
return path
```

Reply:
@FollowTheProcess I made this and the previous suggestions you made. Thanks a lot!